db/migrate/20240218141409_add_alert_columns.rb

```diff
@@ -0,0 +1,6 @@
+class AddAlertColumns < ActiveRecord::Migration[6.1]
+  def change
+    add_column :alerts, :variables, :json
```

Comment:
would jsonb be better?

Reply:
I'm not sure. I think jsonb is slower to write, which could impact regeneration times. Otherwise with this approach it'll be down to whether postgres implementation of json_to_record vs jsonb_to_record has much of a performance difference